Dockerfile

```diff
@@ -1,19 +1,16 @@
 FROM zzrot/alpine-node
 
-RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-# Dev install for build.
-COPY package.json /usr/src/app/package.json
```

Comment:
It's good practice in Nodejs to start copying the package.json that contains the modules to install. So if nothing changed in the modules, but the code has changed (line 9), the `npm install` won't be run.